apps/web/pages/auth/login.tsx

```diff
@@ -51,6 +58,7 @@ export default function Login({
 
   const [twoFactorRequired, setTwoFactorRequired] = useState(false);
   const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [successMessage, setSuccessMessage] = useState<string | null>(null);
```

Comment:
It's always nice to show a success message to the user once everything went well and a redirection is about to happen.